MANUAL_REGRESSION.md

```diff
@@ -399,6 +399,16 @@ On iOS:
     - Upload selfie
     - Confirm
 
+ADDITIONAL TEST (for scenario where integrator sets `requestedVariant: 'video'` with no other options set):
+
+1. In the browser's console, override the initialised config with the following:
+
+```javascript
+  window.onfidoSdkHandle.setOptions({ steps: [{type: 'face', options: { requestedVariant: 'video' }}]})
```

Comment:
If you run this command on a screen different than the welcome, it will break. I think that it might confuse our crowd testers. I suggest to use the querystring `?liveness=true`

Reply:
There is actually already a manual regression test with `?liveness=true` (test 26a) but it doesn't catch the scenario where the bug was reported. 

TBH I'm not sure how to better test for it since it depends on how an integrator initialises the SDK so open to any suggestions, especially if it can be an automated test.

Reply:
As discussed, have updated the manual regression test to use the JSFiddle instead to make it easier for crowd testers.